extensions/google-maps-search/package.json

```diff
@@ -5,6 +5,9 @@
   "description": "Quickly open Google Maps directions, for example from your current location to your home.",
```

Comment:
I'm not sure why `google-maps-search` extension was modified 😅 could you update the branch to make sure your PR only touches the `dev-to` extension?